web_poet/serialization/functions.py

```diff
@@ -0,0 +1,108 @@
+import json
+from typing import Type
+
+from .. import HttpResponse, HttpResponseBody, HttpResponseHeaders, ResponseUrl
+from ..page_inputs.url import _Url
+from .api import (
+    SerializedLeafData,
+    deserialize_leaf,
+    register_serialization,
+    serialize_leaf,
+)
+
+
+def _serialize_dict(o: dict) -> SerializedLeafData:
+    return {"json": json.dumps(o).encode()}
```

Comment:
```suggestion
    return {"json": json.dumps(o, ensure_ascii=False, sort_keys=True).encode()}
```

* ensure_ascii allows to save space, and also makes it more readable
* sort_keys ensures that there won't be spurious differences in key order, e.g. when we re-create something. It's not very likely in recent Python versions, as dict preserve the insert order, but it still might happen.